src/comp_loinc/schema/comp_loinc.yaml

```diff
@@ -181,23 +179,23 @@ classes:
       # for supplementary model slots: https://loinc.org/kb/enriched-linkages-between-loinc-terms-and-loinc-parts/#detailedmodel
 
       supplementary_property:
-        slot_uri: loinc_property:PROPERTY-supplementary
+        slot_uri: loinc_property:PROPERTY
```

Comment:
Ah! Do you think that these changes will fix?:
- https://github.com/loinc/comp-loinc/issues/202#issuecomment-2968192037

Not all the URIs have been changed though in this PR (yet). Is the goal to do that? I'm not sure what to do about "component", because it has one variation/URI in the primary model, but 3 in the supplementary.